docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -415,18 +415,26 @@ spec:
 
 .Verification
 
-* Verify that MariaDB and RabbitMQ are running for all defined cells:
+* Verify that the Galera and RabbitMQ status is `Running` for all defined cells:
 +
+[source,yaml]
 ----
 $ RENAMED_CELLS="cell1 cell2 cell3"
```

Comment:
This may be slightly tangential, but given that the most common use case is not to have multiple cells don't you think we should have a note here to clarify that this should be modified to suit the environment?

Reply:
Yes could be done but it's not related to the bug i'm trying to fix. I could piggy-back another commit to the PR if it was brought up earlier in the sprint but now i'd just focus on the must haves.

The `RENAMED_CELLS` variable is used across the docs in many places, it is described in https://openstack-k8s-operators.github.io/data-plane-adoption/user/downstream.html#adopting-the-compute-service_adopt-control-plane

Now that is searched for it, i see that description is not the first place the variable occurs in the docs, which could be improved upon, but again it's irrelevant to this bug.

Reply:
In the guide, it is stated as an example we stick with to deploy 3 cells further on.
We could surely improve that in some other commits